fast_llm/layers/language_model/head.py

```diff
@@ -113,6 +113,12 @@ def __init__(
             peft=self._peft,
         )
 
+        self._compute_lm_loss = self.config.language_model_loss_factor > 0.0 or self.config.track_language_model_loss
```

Comment:
This and `_compute_distillation_loss` would be more appropriate in the config, ex. as a `cachedproperety`